src/main/java/frc/trigon/robot/subsystems/swerve/Swerve.java

```diff
@@ -88,54 +87,61 @@ public Rotation2d getHeading() {
         return Rotation2d.fromDegrees(SwerveConstants.GYRO.getSignal(Pigeon2Signal.YAW));
     }
 
-    public Translation3d getFieldRelativeVelocity3d() {
-        return new Translation3d(
-                getFieldRelativeVelocity().vxMetersPerSecond,
-                getFieldRelativeVelocity().vyMetersPerSecond,
-                0
-        );
+    public ChassisSpeeds getFieldRelativeChassisSpeeds() {
+        final ChassisSpeeds selfRelativeSpeeds = getSelfRelativeChassisSpeeds();
+        return ChassisSpeeds.fromRobotRelativeSpeeds(selfRelativeSpeeds, RobotContainer.ROBOT_POSE_ESTIMATOR.getEstimatedRobotPose().getRotation());
     }
 
-    public ChassisSpeeds getFieldRelativeVelocity() {
-        final ChassisSpeeds selfRelativeSpeeds = getSelfRelativeVelocity();
-        return ChassisSpeeds.fromRobotRelativeSpeeds(selfRelativeSpeeds, RobotContainer.ROBOT_POSE_ESTIMATOR.getEstimatedRobotPose().getRotation());
+    public Translation2d getFieldRelativeVelocity() {
+        final ChassisSpeeds chassisSpeeds = getFieldRelativeChassisSpeeds();
+        return new Translation2d(chassisSpeeds.vxMetersPerSecond, chassisSpeeds.vyMetersPerSecond);
     }
 
-    public ChassisSpeeds getSelfRelativeVelocity() {
+    public ChassisSpeeds getSelfRelativeChassisSpeeds() {
         return SwerveConstants.KINEMATICS.toChassisSpeeds(getModuleStates());
     }
 
+    public Translation2d getSelfRelativeVelocity() {
```

Comment:
Why would one need this in the translation form? ChassisSpeeds is the standard for swerve